src/main/webapp/app/shared/util/core-submission-shared/core-submission-utils.ts

```diff
@@ -0,0 +1,37 @@
+import { Review } from 'app/shared/model/firebase/firebase.model';
+
+export function useLastReviewedOnly<T>(obj: T, isSavingGene: boolean = false): T | undefined {
+  function useLastReviewedOnlyRec(curObj: T): T | undefined {
+    const newObj: T = {} as T;
+    for (const [key, value] of Object.entries(curObj as Record<string, unknown>)) {
+      if (key.endsWith('_review')) {
+        const temp: Review = Object.assign({} as Review, value);
+        if (!isSavingGene) {
+          delete temp.lastReviewed;
+        }
```

Comment:
Maybe this is copied over from the legacy, but do you know what this is for?

Reply:
It doesn't seem like this is needed. It'll remove it. I can't remember why I put it there in the first place.